components/up.libsonnet

```diff
@@ -64,9 +64,13 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         '--duration=0',
         '--queries-file=/etc/up/queries.yaml',
         '--log.level=debug',
-      ]) + container.withPorts([
+      ] + if std.objectHas(up.config, 'writeEndpoint') then [
```

Comment:
If we are going down to this path, let's make the read endpoint optional as well.

Reply:
added